frontend/js/components/TableActiveRows.js

```diff
@@ -0,0 +1,42 @@
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+import { fetchActiveBattles } from '../actions/setBattle';
+import { selectBattlesResult, selectBattles } from '../utils/selectors';
+
+import ActiveRow from './ActiveRow';
+
+class TableActiveRows extends Component {
+  componentDidMount() {
+    const { fetchActiveBattles } = this.props;
+    fetchActiveBattles();
+  }
+
+  render() {
+    const { battlesResult, battleList } = this.props;
+    if (!battlesResult) return <>Loading</>;
+    return battlesResult.map((id) => {
+      const battle = battleList[id];
+      return <ActiveRow key={id} battle={battle} />;
```

Comment:
This component shouldn't receive battleList, just pass the battleID to the child component and the child component will fetch the battle from state.